ts/DataGrid/DataGrid2/DataGridOptions.d.ts

```diff
@@ -158,7 +159,8 @@ export interface ColumnOptions {
     headFormat?: string;
 
     /**
-     * Extendable method for formatting each cell's in DataGrid.
+     * Extendable method for formatting cell's within the given column of
```

Comment:
IMO it should be phrased as: 
`Callback function for formatting cells within the given column of the DataGrid.`